source/frontends/sdl/imgui/sdldebugger.cpp

```diff
@@ -275,10 +275,9 @@ namespace sa2
       ImGui::TableSetupColumn("Opcode", 0, 8);
       ImGui::TableSetupColumn("Symbol", 0, 10);
       ImGui::TableSetupColumn("Disassembly", 0, 20);
-      ImGui::TableSetupColumn("Target", 0, 4);
-      ImGui::TableSetupColumn("Value", 0, 6);
-      ImGui::TableSetupColumn("Immediate", 0, 4);
-      ImGui::TableSetupColumn("Branch", 0, 4);
+      ImGui::TableSetupColumn("Target", 0, 7);
+      ImGui::TableSetupColumn("Branch", 0, 1);
```

Comment:
I prefer the sizes to be 4 3 4 and you need to reduce the number of columns to 9

Reply:
Sorry, what do you mean by 4 3 4? I combined the Target, Value, and Immediate columns into one because that's what AppleWin does. Do you still prefer all of them in separate columns?

Will fix the number of columns.